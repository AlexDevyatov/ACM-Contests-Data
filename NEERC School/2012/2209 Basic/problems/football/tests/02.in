100
4:5
4:2
2:6
9:7
2:8
9:9
8:7
6:7
4:9
6:6
4:7
1:9
5:1
4:0
9:2
2:0
5:6
0:9
5:7
7:5
6:9
0:5
0:0
4:6
7:7
8:5
9:8
6:0
1:1
1:4
4:3
8:6
5:0
1:2
5:9
4:8
4:4
8:4
0:4
5:3
7:4
6:1
8:1
2:1
8:9
4:1
8:0
3:2
3:0
0:6
7:2
2:4
0:3
9:0
6:5
1:8
7:8
8:3
8:8
1:0
1:7
5:5
3:8
0:7
0:1
7:3
3:5
3:1
3:7
1:3
2:9
2:2
7:6
8:2
3:6
3:3
2:5
2:7
3:9
5:8
6:8
0:8
2:3
5:4
3:4
6:2
7:9
6:3
5:2
9:6
6:4
1:6
1:5
7:0
7:1
9:5
9:3
9:1
9:4
0:2
