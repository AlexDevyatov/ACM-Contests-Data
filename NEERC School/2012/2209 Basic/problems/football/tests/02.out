40 59
69 30
18 81
64 35
7 92
45 54
56 43
38 61
10 89
48 51
23 76
1 98
83 16
84 15
94 5
71 28
39 60
0 99
30 69
66 33
21 78
14 85
54 45
31 68
47 52
73 26
55 44
93 6
53 46
26 73
60 39
65 34
89 10
43 56
15 84
16 83
50 49
80 19
20 79
68 31
74 25
88 11
95 4
62 37
36 63
77 22
98 1
61 38
78 21
9 90
87 12
33 66
27 72
99 0
58 41
4 95
37 62
86 13
46 53
63 36
8 91
49 50
11 88
5 94
44 55
81 18
32 67
70 29
17 82
34 65
3 96
52 47
57 42
91 8
24 75
51 48
25 74
12 87
6 93
22 77
29 70
2 97
42 57
59 40
41 58
82 17
28 71
75 24
76 23
72 27
67 32
13 86
19 80
96 3
92 7
79 20
90 9
97 2
85 14
35 64
